sim/midas/src/main/scala/midas/models/dram/Util.scala

```diff
@@ -349,6 +371,51 @@ class AXI4Releaser(implicit p: Parameters) extends Module {
   io.egressResp.bReady := io.b.ready
 }
 
+class FIFOAlignedAddressMatcher(val entries: Int, addrWidth: Int, beatBytes: Int, numBeats: Int) 
```

Comment:
Add Scaladoc for this explaining each of the parameters.

Reply:
Added comments